app/scripts/controllers/reviewRegistration.js

```diff
@@ -249,8 +249,13 @@ angular.module('confRegistrationWebApp')
     $scope.mergeAndConfirmRegistration = function () {
       // Pay for the spouse's registration before merging it with the spouse
       $scope.submittingRegistration = true;
-      registration.submitPayment($scope.currentPayment, $scope.spouseRegistration, $scope.acceptedPaymentMethods()).then(function () {
-        return registration.mergeWithSpouse(currentRegistration, $scope.spouseRegistration);
+
```

Comment:
@dougworner  There is no existing payment to delete as we have not yet submitted one for the current registrant.